clients/src/main/java/org/apache/kafka/clients/CommonClientConfigs.java

```diff
@@ -50,7 +50,7 @@
     public static final String RECONNECT_BACKOFF_MS_DOC = "The amount of time to wait before attempting to reconnect to a given host. This avoids repeatedly connecting to a host in a tight loop. This backoff applies to all requests sent by the consumer to the broker.";
 
     public static final String RETRY_BACKOFF_MS_CONFIG = "retry.backoff.ms";
-    public static final String RETRY_BACKOFF_MS_DOC = "The amount of time to wait before attempting to retry a failed fetch request to a given topic partition. This avoids repeated fetching-and-failing in a tight loop.";
+    public static final String RETRY_BACKOFF_MS_DOC = "The amount of time to wait before attempting to retry a failed request to a given topic partition. This avoids repeated failing of requests in a tight loop.";
```

Comment:
I think I would say the second sentence as:

"This avoids repeatedly sending requests in a tight loop under some failure scenarios."

As it's phrased, it could be interpreted as meaning that we would repeatedly fail the same request.